src/entries/popup/pages/settings/customRPC/customRPC.tsx

```diff
@@ -0,0 +1,249 @@
+                      borderColor="accent"
+                      onClick={() =>
+                        setActiveRPC({
+                          rpcUrl: customRPC.rpcUrl,
+                          chainId: customRPC.chainId,
+                        })
+                      }
+                      selected={chain.activeRpcUrl === customRPC.rpcUrl}
+                    />
+                  </Inline>
+                  <Inline alignHorizontal="right">
+                    <Button
+                      onClick={() =>
+                        removeCustomRPC({ rpcUrl: customRPC.rpcUrl })
+                      }
+                      color="accent"
+                      height="36px"
+                      variant="raised"
+                    >
+                      Remove
+                    </Button>
+                  </Inline>
+                </Stack>
+              </Box>
+            );
+          })}
+        </Stack>
+
+        {customRPCAssetsForChain?.map((asset, i) => (
+          <Box
+            background="surfaceSecondaryElevated"
+            borderRadius="16px"
+            boxShadow="12px"
+            width="full"
+            padding="16px"
+            key={i}
+          >
+            <Stack space="4px">
+              <Text
+                align="left"
+                weight="semibold"
+                size="9pt"
+                color="labelSecondary"
+              >
+                {asset.address}
+              </Text>
+              <Text
+                align="left"
+                weight="semibold"
+                size="12pt"
+                color="labelSecondary"
+              >
+                {asset.decimals}
+              </Text>
+              <Text
+                align="left"
+                weight="semibold"
+                size="12pt"
+                color="labelSecondary"
+              >
+                {asset.symbol}
+              </Text>
+            </Stack>
+            <Inline alignHorizontal="right">
+              <Button
+                onClick={() =>
+                  removeCustomRPCAsset({ address: asset.address, chainId })
+                }
+                color="accent"
+                height="24px"
+                variant="raised"
+              >
+                Remove Asset
+              </Button>
+            </Inline>
+          </Box>
+        ))}
+
+        <Box
+          background="surfaceSecondaryElevated"
+          borderRadius="16px"
+          boxShadow="12px"
+          width="full"
+          padding="16px"
+        >
+          <Stack space="8px">
+            <Input
+              onChange={(t) =>
+                onInputChange<string>(t.target.value, 'string', 'address')
+              }
+              height="32px"
+              placeholder="Address"
+              variant="surface"
+              value={asset.address}
+              // onBlur={onRpcUrlBlur}
+              // borderColor={validations.rpcUrl ? 'accent' : 'red'}
```

Comment:
I assume this code is commented bc this a temporary UI right?

Reply:
will remove